src/radioButtonGroup/index.js

```diff
@@ -13,12 +13,14 @@ class RadioButtonGroup extends React.Component {
     };
   }
 
-  handleCheckListChange = (currentlyActive) => {
-    console.log(currentlyActive);
+  handleCheckListChange = (e, option) => {
```

Comment:
This should be handleRadioBtnChange. This should change on every occurance

Reply:
Updated.